Modules/Scripted/DICOMPlugins/DICOMSlicerDataBundlePlugin.py

```diff
@@ -65,6 +65,7 @@ def examineFiles(self,files):
         loadable.name = name + ' - as Slicer Scene'
         loadable.selected = True
         loadable.tooltip = 'Appears to contain a slicer scene'
```

Comment:
If we are confident that it is a Slicer scene (I think we should be fairly certain) then the tooltip text should be a bit more confident, too